setup.py

```diff
@@ -163,6 +163,7 @@ def build_extension(self, ext):
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.9",
     ],
+    python_requires=">= 3.6",
```

Comment:
I think you'll also want to remove the `Python :: 2.7` from the list of classifiers above.